files/zh-cn/web/http/status/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: HTTP 响应代码
+slug: Web/HTTP/Status
+tags:
+  - HTTP
+  - 状态码
+translation_of: Web/HTTP/Status
+---
+{{HTTPSidebar}}
+
+HTTP 响应状态码用来表明特定 [HTTP](/en-US/docs/Web/HTTP) 请求是否成功完成。
```

Comment:
just using the link to `zh-CN` (even if the document does not exist, mdn will fallback to `en-US` for not existing document).

```markdown
[HTTP](/zh-CN/docs/Web/HTTP)
```